ofrak_patch_maker/ofrak_patch_maker/toolchain/llvm_12.py

```diff
@@ -115,6 +115,17 @@ def __init__(
     def name(self) -> str:
         return "LLVM_12_0_1"
 
+    def keep_section(self, section_name: str):
+        if section_name in self._linker_keep_list:
+            return True
+        if self._config.separate_data_sections:
+            for keep_section in self._linker_keep_list:
```

Comment:
Should this be a more restricted list, limited to just relevant data section prefixes? I think it would be pretty weird if some `.text.blahblah` section is now included if the separate _data_ sections flag is enabled. This should probably only pass if the section name starts with ".rel", maybe ".data" or ".rodata"?

Reply:
Or, @EdwardLarson with the expectation that we will likely want to do something similar with .text at some point... should we make the toolchain config flag more general, i.e. something like `config.wildcard_standard_sections`?

Reply:
That seems fine to me, though maybe an even more functional-sounding name. Some ideas/examples:
`keep_standard_prefixed_sections`
`keep_subsections`
`include_subsections`

But `separate_data_sections` also does cause compiler flags to be added when building object files - are these two separate behaviors that it makes sense to toggle individually? Does having separate data sections always necessarily mean you want to keep sections starting with a standard prefix? It seems like the reverse is not true.